### DISCUSSION TITLE
Εμβόλιμη εξεταστική ΧΕΙΜ 2024-2025

### DISCUSSION THREAD

**Discussion by panosath** [courses-ionio/sw#1787] [Q&A] [answered]
Καλησπέρα, σχετικά με την εμβόλιμη εξεταστική του Ιαν-Φεβ 2025, ισχύει ό,τι αναφέρεται στο [teaching - Απαντήσεις σε συχνές απορίες](https://epidrome.github.io/teaching/deliverables/), σχετικά με τους επι πτυχίω φοιτητές; Ρωτάω καθώς σήμερα, 15 Νοεμβρίου 2025 έγινε γνωστό το ότι είναι δυνατή η δήλωση μαθημάτων χειμερινού εξαμήνου. Επομένως μπορώ από σήμερα να ξεκινήσω την πραγματοποίηση των παραδοτέων; Ευχαριστώ.

**Comment by epidrome** [comment, marked as answer]
το μάθημα αυτό είναι ελεύθερης επιλογής, και όπως λένε οι οδηγίες και έχεις ενημερωθεί ήδη 

https://github.com/courses-ionio/sw/discussions/1786

απαιτεί συμμετοχή σε δύο τουλάχιστον προφορικές παρουσιάσεις στην κανονική του διάρκεια
